tests/unit/amo/components/TestAddon.js

```diff
@@ -866,34 +867,47 @@ describe(__filename, () => {
   });
 
   it('renders a static theme preview as an image', () => {
+    const headerImageThumb = 'https://addons.cdn.mozilla.net/thumb/54321.png';
+    const newPreview = [
+      {
+        ...fakePreview,
+        thumbnail_url: headerImageThumb,
+      },
+    ];
+
     const root = shallowRender({
       addon: createInternalAddon({
         ...fakeTheme,
         type: ADDON_TYPE_STATIC_THEME,
+        previews: newPreview,
       }),
     });
     const image = root.find('.Addon-theme-header-image');
     expect(image.type()).toEqual('img');
     expect(image).toHaveClassName('Addon-theme-header-image');
-    expect(image.prop('src')).toEqual(
-      'https://addons.cdn.mozilla.net/123/image.png',
-    );
+    expect(image.prop('src')).toEqual(headerImageThumb);
     expect(image.prop('alt')).toEqual('Preview of Dancing Daisies by MaDonna');
   });
 
   it('renders the preview image from the previews array if it exists for the lightweight theme', () => {
+    const headerImageThumb = 'https://addons.cdn.mozilla.net/thumb/12345.png';
+    const newPreview = [
```

Comment:
You don't really need this `newPreview` variable - you could just inline this into the call to `shallowRender`.